.github/workflows/ci.yml

```diff
@@ -811,6 +811,12 @@ jobs:
               --benchmark_files="${EXECUTION_BENCHMARK_RESULTS_DIR}/benchmark-results-c2-standard-16.json" \
               --benchmark_files="${EXECUTION_BENCHMARK_RESULTS_DIR}/benchmark-results-a2-highgpu-1g.json" \
               --compile_stats_files="${COMPILE_STATS_RESULTS}"
+      - name: Trigger DB import
+        continue-on-error: true
+        env:
+          DB_IMPORTER_TYPE: ${{ github.event_name == 'pull_request' && 'presubmit' || 'postsubmit' }}
```

Comment:
In other places we use `fromJson(needs.setup.outputs.is-pr)` to check if the run is on PR or postsubmit (including manual workflow_dispatch). Maybe we can check on the same thing here?

Reply:
Yeah, great. Added. Thanks.